### DISCUSSION TITLE
Failed

### DISCUSSION THREAD

**Discussion by sultanmonsur** [anddea/revanced-patches#647] [General]
When will resolve?
Lately this problem is coming up after every update of yours.
![Screenshot_2024-06-24-11-36-20-02_84d3000e3f4017145260f7618db1d683](https://github.com/anddea/revanced-patches/assets/61897416/49dbcbc2-ecc2-4880-a58a-f87f84704068)

**Comment by anddea** [comment]
There are open issues for this, do not create spammy duplicates
